CMakeLists.txt

```diff
@@ -447,16 +447,18 @@ if (WIN32 OR is_xcode)
   group_sources(Builds)
 endif()
 
-if(unity)
+if (NOT fuzzer_conditions)
```

Comment:
I don't have a strong opinion, but why not just ahead a allow `rippled` to be a target? You can still just build the fuzzed exes by name if you don't want to build `rippled`  e.g. `make fulfillment_fuzzer`.